features/post/post-card/post-info-section/post-info-section.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { ReactNode } from 'react';
+
+import { BookmarkOutline } from '@/assets/icons/bookmark-outline';
+import { HeartFilled } from '@/assets/icons/heart-filled';
+import { HeartOutlined } from '@/assets/icons/heart-outlined';
+import { PaperPlaneOutline } from '@/assets/icons/paper-plane-outline';
+import { DateFormatter, useDateFormat } from '@/shared/hooks/useDateFormat';
+import { useTranslation } from '@/shared/hooks/useTranslation';
+import { Avatar, IconButton, Typography } from '@/shared/ui';
+import { useRouter } from 'next/router';
+
+import s from './post-info.module.scss';
+
+type PostInfoProps = {
+  actions?: ReactNode;
+  createdAt: string;
+  icons?: {
+    avatar: string;
+    userName: string;
+  }[];
+  likesCount: number;
+};
+
+export const PostInfoSection = (props: PostInfoProps) => {
+  const { actions, createdAt, icons, likesCount } = props;
+  const { locale } = useRouter();
+  const { t } = useTranslation();
+
+  const customDateTimeFormatter: DateFormatter = parts => {
+    const date = new Date(createdAt);
+
+    return new Intl.DateTimeFormat(locale, {
+      day: 'numeric',
+      month: 'long',
+      year: 'numeric',
+    }).format(date);
+  };
+
+  const formattedDate = useDateFormat(
+    { day: 'numeric', month: 'long', year: 'numeric' },
+    customDateTimeFormatter
+  );
+
+  const formatNumberWithSpaces = (num: number | string): string => {
+    return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ' ');
+  };
+
+  const getInitials = (userName: string): string => {
+    const words = userName.split(' ');
+    const initials = words.map(word => word.charAt(0).toUpperCase());
+
+    return initials.join('');
+  };
+
+  return (
+    <div className={s.root}>
+      {actions}
+      <div className={s.postMetrics}>
+        <div className={s.lickedCount}>
+          <div className={s.icons}>
+            {icons
+              ?.slice(0, 3)
+              .map((item, index) => (
+                <Avatar
+                  classes={{ avatarRoot: s.icon }}
+                  fallback={getInitials(item.userName)}
+                  key={index}
+                  size={'small'}
+                  src={item.avatar}
+                />
+              ))}
+          </div>
+          <Typography.Regular14>
+            {`${formatNumberWithSpaces(likesCount)}`}
```

Comment:
`Intl.PluralRules` is needed here to display the plural of the word “Like” and its Russian counterpart. [Link to explore the topic](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules#examples). Perhaps the plural rule map for the right words should be placed in the locale files (assuming you can rewrite its typing correctly). If this causes problems, don't touch the locale objects, but create the maps directly in the component.